### ISSUE TITLE
how to get get train-acc and validation-acc per epoch explicitly per epoch?

### ISSUE THREAD

**Issue by Royzon** [apache/incubator-mxnet#15534]
how can I get train-acc and validation-acc explicitly rather than just call modul.fit()， or In other words, how can I customize the call_back to get train-acc and validation-acc per epoch?

**Comment by braindotai** [comment]
You can pass evaluation data in .fit call as below:

```python
# create a module
mod = mx.mod.Module(symbol=net,
                    context=mx.gpu(),
                    data_names=['data'],
                    label_names=['softmax_label'])

# fit the module
mod.fit(train_iter,
        eval_data=val_iter,
        optimizer='sgd',
        optimizer_params={'learning_rate':0.1},
        eval_metric='acc',
        num_epoch=7)
```
For more info checkout [here](https://mxnet.incubator.apache.org/versions/master/tutorials/basic/module.html)

**Comment by braindotai** [comment]
You can pass eval_data in the `.fit` call method as below:
```python
# create a module
mod = mx.mod.Module(symbol=net,
                    context=mx.cpu(),
                    data_names=['data'],
                    label_names=['softmax_label'])

# fit the module
mod.fit(train_iter,
        eval_data=val_iter,
        optimizer='sgd',
        optimizer_params={'learning_rate':0.1},
        eval_metric='acc',
        num_epoch=7)
```
Will print
```
INFO:root:Epoch[0] Train-accuracy=0.325437
INFO:root:Epoch[0] Time cost=0.550
INFO:root:Epoch[0] Validation-accuracy=0.568500
INFO:root:Epoch[1] Train-accuracy=0.622188
INFO:root:Epoch[1] Time cost=0.552
INFO:root:Epoch[1] Validation-accuracy=0.656500
```

**Comment by frankfliu** [comment]
@mxnet-label-bot add [python, question]
